glad/parse.py

```diff
@@ -732,6 +747,15 @@ def select(self, api, version, profile, extension_names, sink=LoggingSink(__name
         extensions = [self.extensions[api][name] for name in extension_names
                       if name in self.extensions[api]]
 
+        # Load documentation for the specific configuration
+        if config['WITH_DOCS']:
```

Comment:
Seems like the specification doesn't need the docs at all. Instead of tying the docs to the specification, the templating could be the part where the specification gets married with the docs.

Reply:
I am not quite sure about how I'd like it to look at the end, don't worry too much about it, it's easy for me to just move everything a bit around.

But basically in mind right now the docs are a separate source of data independent of the spec and you just pass it as some kind of "docs db" into the template and the template tries to lookup the docs from the db when rendering.

What guarantuess you get the correct symbols? Either it's in the template, a similar mechanism to `spec.select()` or just nothing, if you pass the wrong docs then there are just no docs for the requested symbols.

But again don't worry too much about that, these are the things that are easy to change later, the hard part is the actual logic!

Reply:
I moved this logic and now docs are loaded separately, only for the specifications listed by the user. Then, those loaded docs sources are "selected" from a feature_set, and passed directly to the template generator function, without depending on the spec. I think I ended up understanding what you meant, but if not, feel free to change it later. I'll be working on using the Khronos Refpages as source :-)